README.md

```diff
@@ -1,3 +1,17 @@
+# Execução
+
+[Será necessário que possua o Docker instalado em seu sistema operacional para executar esse projeto.](https://docs.docker.com/get-docker/)
+
+Após a instalação:
+
+`docker compose build`
+
+`docker network create reverse-proxy`
+
+`docker network create mysql`
```

Comment:
Really want this process executed automatically but cannot find an effective solution. If you reviewer know a way to make it please leave a suggestion.